ledger-core/v2/configuration/dummyapi.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package configuration
+
+// APIRunner holds configuration for api
+type TestWalletAPI struct {
```

Comment:
Maybe it's better to rename the file, too.

Reply:
agree

Reply:
yes, done